x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/parser/EscapedFunctionsTests.java

```diff
@@ -62,6 +62,40 @@ private Literal timestampLiteral(String date) {
         return (Literal) exp;
     }
 
+    private String buildDate() {
+        StringBuilder sb = new StringBuilder();
+        int length = randomIntBetween(4, 9);
+
+        if (randomBoolean()) {
+            sb.append('-');
```

Comment:
I don't understand this. Why is the date starting with `-`?

Reply:
To test negative Year.